src/components/RightBar/rightbar.styles.js

```diff
@@ -16,6 +16,13 @@ export const RightBarContainer = styled.div`
   section {
     display: flex;
     flex-direction: column;
+
+    .learn {
+      color: #000000;
+      font-weight: 300;
+      margin-bottom: 20px;
+      padding-left: 3px;
+    }
   }
 
   @media (max-width: 1200px) {
```

Comment:
@ogwurujohnson  can you change this to 1380px because for all screens less than that. The sidebar is overlapping the cards. Aaron change it here. I am not sure why it isn't showing up in diff https://github.com/bravolabs/bravo-fe/pull/55/files